REQUIRE

```diff
@@ -1,2 +1,4 @@
 julia 0.7.0-DEV.5190 # for cholesky function
 Compat 0.61.0
+SuiteSparse
```

Comment:
So currently we're in a funny situation where SuiteSparse is a registered package, yet we don't want to use that, we want to use SuiteSparse from the stdlib. So `using SuiteSparse` is good, but it should be removed from REQUIRE.